src/Controllers/HomeController.php

```
@@ -78,11 +97,18 @@ public function index(): void
-            } catch (\Throwable $e) {
-                $jiraProjects = [];
+            if (filter_var($this->config['jira']['home_project_picker_live_fetch'] ?? false, FILTER_VALIDATE_BOOLEAN)) {
+                try {
+                    $jiraService  = ($this->jiraServiceFactory)($this->config['jira'] ?? [], $integration, $this->db);
+                    $jiraProjects = $jiraService->getProjects();
+                } catch (\Throwable $e) {
+                    Logger::warn('[StratFlow] Home Jira project fetch failed', [
+                        'org_id' => $orgId,
+                        'integration_id' => (int) ($integration['id'] ?? 0),
+                        'error' => $e->getMessage(),
+                    ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Services\Logger' in method 'index'. (undefined)

(StaticAccess)